src/libraries/System.Formats.Tar/tests/TarFile/TarFile.CreateFromDirectory.File.Tests.cs

```diff
@@ -225,6 +225,45 @@ public void SkipRecursionIntoDirectorySymlinks()
             Assert.Null(reader.GetNextEntry()); // file.txt should not be found
         }
 
+        [ConditionalFact(typeof(MountHelper), nameof(MountHelper.CanCreateSymbolicLinks))]
+        public void WriteSymlinkTargetsInsideTheArchiveRoundtrips()
```

Comment:
nit:

I assume you didn't add this to [`./tests/TarWriter/TarWriter.WriteEntry.Entry.Roundtrip.Tests.cs`](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Formats.Tar/tests/TarWriter/TarWriter.WriteEntry.Entry.Roundtrip.Tests.cs) because there are only `TarWriter` tests there. 

Could you please create a similar file in this folder, say `TarFile.CreateFromDirectory.File.Roundtrip.cs`?

Reply:
Sure, I added it next to the existing test (we have only a few (two) existing tests related to symlinks).